docroot/sites/all/modules/dev/ws_rules_features/modules/positive_feedback_rules/positive_feedback_rules.rules_defaults.inc

```diff
@@ -22,16 +22,16 @@ function positive_feedback_rules_default_rules_configuration() {
       ],
       "DO" : [
         { "mail" : {
-            "to" : "[node:field_member_i_trust:mail]",
-            "subject" : "[Warmshowers.org] You have received member feedback",
+            "to" : [ "node:field-member-i-trust:mail" ],
```

Comment:
Surely that can't be right, the token is [node:field-member-i-trust:mail], not with quote marks in it??? I think maybe it was just the hyphens that needed to change.